### PR TITLE
[fix] fix typo

### PR DESCRIPTION
**Pull request by cmdbug**

### CODE REVIEW

File: source/device/cpu/op/softmax/softmax_ref.c [hunk 1]
```diff
@@ -57,7 +57,7 @@ static int run(struct node_ops* node_ops, struct exec_node* exec_node, struct ex
 
     struct softmax_param* softmax_param = (struct softmax_param*)ir_node->op.param_mem;
 
-    // Check: axis must be in the range: [-input_tensor->dim_num, input_tensor->dim_num)
+    // Check: axis must be in the range: [-input_tensor->dim_num, input_tensor->dim_num]
```
Comment (nullptr-leo) [inline source/device/cpu/op/softmax/softmax_ref.c]: `[)`

Comment (cmdbug) [reply source/device/cpu/op/softmax/softmax_ref.c]: It has been changed.

File: source/device/cpu/op/softmax/softmax_ref.c [hunk 2]
```diff
@@ -57,7 +57,7 @@ static int run(struct node_ops* node_ops, struct exec_node* exec_node, struct ex
 
     struct softmax_param* softmax_param = (struct softmax_param*)ir_node->op.param_mem;
 
-    // Check: axis must be in the range: [-input_tensor->dim_num, input_tensor->dim_num)
+    // Check: axis must be in the range: [input_tensor->dim_num, -input_tensor->dim_num)
```
Comment (nullptr-leo) [inline source/device/cpu/op/softmax/softmax_ref.c]: 原来对的吧~

Comment (cmdbug) [reply source/device/cpu/op/softmax/softmax_ref.c]: 蒙蔽了，if 里面是 return 出去了。qwq

**Review by BUG1989 — APPROVED** [review]
lgtm

**Pull request merged**